api/src/models/users.js

```diff
@@ -1,4 +1,5 @@
 const connection = require('../db/connection')
+const { validateRole } = require('../utils/roles')
 
 class Users {
```

Comment:
To add a role, do we just call `update`? Do we want to make it more explicit than that?